### ISSUE TITLE
Builder fails to parse declaration `const char* const A("s");`

### ISSUE THREAD

**Issue by jpsacha** [bytedeco/javacpp#179]
Basler Pylon library 5.0.5 is using declaration 

```cpp
const char* const ManufacturerInfoKey("ManufacturerInfo");
```

JavaCPP Builder fails while parsing header file with that line. Complete header file is attached. The actual error message:

```
Failed to execute goal org.bytedeco:javacpp:1.3.3-SNAPSHOT:build (javacpp.parser) on project pylon: 
Failed to execute JavaCPP Builder: C:\Program Files\Basler\pylon 5\Development\include\pylon\DeviceInfo.h:50: 
Could not parse declaration at '='
```

[DeviceInfo.h.zip](https://github.com/bytedeco/javacpp/files/981021/DeviceInfo.h.zip)

**Comment by saudet** [comment]
This should be fixed with the commit above. Let me know if you still have issues! Thanks

**Comment by jpsacha** [comment]
Unfortunately I am still getting the same error with latest (5/14) javacpp.

**Comment by saudet** [comment]
Might be because of `PYLONBASE_API`. Is it properly "ignored"?

**Comment by jpsacha** [comment]
It is ignored with
```java
new Info("PYLONBASE_API").cppTypes().annotations().cppText("")
```

Current version of presets on branch [wip/pylon](https://github.com/jpsacha/javacpp-presets/tree/wip/pylon/pylon/src/main/java/org/bytedeco/javacpp/presets)

**Comment by saudet** [comment]
Thanks, it's fixed now! The issue was only happening with `Info.skip()`.

**Comment by saudet** [comment]
Fix released in version 1.3.3. Thanks for reporting!
